modules/fee-market/src/lib.rs

```diff
@@ -578,7 +578,7 @@ impl<T: Config<I>, I: 'static> Pallet<T, I> {
 
 /// The assigned relayers slash trait
 pub trait Slasher<T: Config<I>, I: 'static> {
-	fn cal_slash_amount(
+	fn calc_amount(
```

Comment:
`slash_amount` sounds a little bit weird to me.

How about `slash_value`?

Reply:
> slash_amount sounds a little bit weird to me.

Have you updated your PR to the latest version? This function named `calc_amount` now.